advanced/editor-command-identifiers.md

```diff
@@ -97,6 +97,7 @@ The commands on the following table are provided by the {{site.productname}} edi
 | mceSelectNode        | Selects a node in the editor. The target node is passed as the value (_`<DOM_node>`_).                                                                                                                                                                                                      |
 | mceSelectNodeDepth   | Selects the parent DOM node 'n' levels above the current node.                                                                                                                                                                                                                              |
 | mceRemoveNode        | Removes the current node or the target node passed as the value (_`<DOM_node>`_).                                                                                                                                                                                                           |
+| mceFocus             | Focuses and activates the editor. The optional value passed in specifies if the editor should only be activated and not focused.                                                                                                                                                            |
```

Comment:
I wonder if we should define (or link to the definition of) activating the editor?

Reply:
Maybe?
```suggestion
| mceFocus             | Focuses and activates the editor. The optional boolean passed in specifies if the editor should only be activated and not focused.                                                                                                                                                            |
```

Reply:
I'm also not sure which way around the boolean goes from reading this. Is it:

> I passed in true because "the editor should only be activated and not focused"

as you've written here, or is it

> I passed in true because the answer to "should it focus as well as activate" is yes

If its the former, maybe write something like
```suggestion
| mceFocus             | Focuses and activates the editor. Optionally pass in `true` if the editor should only be activated, and not focused.                                                                                                                                                            |
```

If the later, maybe
```suggestion
| mceFocus             | Focuses and activates the editor. The optional boolean passed in specifies if the editor should be focused, instead of just activated.                                                                                                                                                            |
```

Or maybe some combination of the two. Either way, maybe we could
* Say what the default is if you don't pass in the optional argument
* Have two code examples below (one with the argument, one using the default, and comment next to them saying what each will do)

Reply:
The "doesn't activate path" shouldn't be documented anyways, as it's internal only. If some only wants to activate an editor the supported way is via `editorManager.setActive`.

Reply:
I was thinking about whether I needed to mention the activate part when writing it. The second sentence can be removed then. 

If we need to elaborate on what activating means, I guess we could reword the description from or link to https://www.tiny.cloud/docs/api/tinymce/tinymce.editor/#focus?

Maybe this could suffice as the description?
```
Focuses and activates the editor. Sets the editor as the activeEditor in the tinymce collection and also places DOM focus inside the editor.
```